Libraries/test/EventsTests.Shared/EventTests.cs

```diff
@@ -754,6 +754,35 @@ private static void Handle(SQSEvent sqsEvent)
             }
         }
 
+        [Theory]
+        [InlineData(typeof(JsonSerializer))]
+#if NETCOREAPP3_1_OR_GREATER        
+        [InlineData(typeof(Amazon.Lambda.Serialization.SystemTextJson.LambdaJsonSerializer))]
+        [InlineData(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
+#endif
+        public void SQSBatchResponseTest(Type serializerType)
+        {
+            var serializer = Activator.CreateInstance(serializerType) as ILambdaSerializer;
+            using (var fileStream = LoadJsonTestFile("sqs-response.json"))
+            {
+                var sqsBatchResponse = serializer.Deserialize<SQSBatchResponse>(fileStream);
```

Comment:
Since this is a response the test has to be written in reverse. Create a instance of `SQSBatchResponse` and then serialize it to JSON to and make sure the JSON is correct, especially the casing.

Reply:
I've updated the test following patterns observed in other tests.